### DISCUSSION TITLE
Praise you

### DISCUSSION THREAD

**Discussion by andredmelo** [nedragrevev/custom-subs#8] [General]
I first of all want to praise this work!
I’ve been looking for accurately translated subtitles for Ghibli movies for a long time, and finally someone is doing an amazing work.
Can I beg you to continue this amazing work with other titles? “Spirited Away”? “Princess Mononoke”? “Howl’s Moving Castle”? “Kiki’s Delivery Service”? “Porco Rosso”? “Ponyo”? “The Wind Rises”?
Can a regular guy help with something?
I’d happily give at least a cup of coffee for each movie :)
O-tsukaresama desu 🙌

**Comment by nedragrevev** [comment]
Thanks for your kind comments and offer to help and/or a coffee! I'm glad you enjoyed the three Ghibli subs.

When I started subbing, I had plans to go through almost all of the Ghibli movies, retranslating and subbing them properly, but I just don't have the time and energy for it anymore, so it's not something I'm looking to get back into. I'm really sorry!
